masschallenge/django-accelerator
# [AC-5989] Programs have a way to make distinctions about if they are included in alumni programs or not

#### Changes introduced in [AC-5989](https://masschallenge.atlassian.net/browse/AC-5989):
- Add a boolean field to the BaseProgram model to represent alumni eligibility that defaults to False

#### How to test
- Unit test added for the change
- Pull this branch and switch to it
- Run accelerate and go to the url `http://localhost:8181/admin/mc/program/{program_id}/change/`. You should see a checkbox field to with the label `Alumni eligible program`

#### Screenshots
<img width="546" alt="screen shot 2018-09-26 at 21 36 43" src="https://user-images.githubusercontent.com/13675851/46118391-4fbfd680-c1d4-11e8-886c-074333bf1b47.png">
